maven-spy/src/main/java/org/jenkinsci/plugins/pipeline/maven/eventspy/reporter/FileMavenEventReporter.java

```diff
@@ -41,13 +41,15 @@
 import java.text.SimpleDateFormat;
 import java.util.Date;
 
+import javax.annotation.concurrent.ThreadSafe;
+
 /**
  * @author <a href="mailto:cleclerc@cloudbees.com">Cyrille Le Clerc</a>
  */
+@ThreadSafe
 public class FileMavenEventReporter implements MavenEventReporter {
     File outFile;
```

Comment:
I think you should add `@GuardedBy` annotation to  the class variables, to make sure FindBugs does the right checks.